worlds/stardew_valley/__init__.py

```diff
@@ -387,8 +204,9 @@ def fill_slot_data(self) -> Dict[str, Any]:
         slot_data.update({
             "seed": self.random.randrange(1000000000),  # Seed should be max 9 digits
             "randomized_entrances": self.randomized_entrances,
-            "modified_bundles": bundles,
+            "modified_bundles": "Junimos are like dogs, they'll eat what you give them, don't sweat it.",
             "client_version": "5.0.0",
+            "Difficulty": "Good Luck"
```

Comment:
Is there a reason for the capital D? All the other entries are in snake_case...

Reply:
We want the D